bench_runner/benchmark_definitions.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+
+
+import dataclasses
+import hashlib
+from pathlib import Path
+
+
+from . import git
+
+
+@dataclasses.dataclass
+class BenchmarkRepo:
+    hash: str
+    url: str
+    dirname: str
+
+
+BENCHMARK_REPOS = [
+    BenchmarkRepo(
+        "56d12a8fd7cc1432835965d374929bfa7f6f7a07",
+        "https://github.com/mdboom/pyperformance.git",
```

Comment:
Why are you pointing to your own fork?

Reply:
Oh, this is just an artifact of needing some modifications to pyperformance early on.  There's no need for this anymore, I'll change it.